packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.kt

```diff
@@ -183,7 +185,7 @@ internal class ImageLoaderModule : NativeImageLoaderAndroidSpec, LifecycleEventL
       promise.reject(ERROR_INVALID_URI, "Cannot prefetch an image for an empty URI")
       return
     }
-    val uri = Uri.parse(uriString)
+    val uri = uriString.toUri()
```

Comment:
Can you amend this + all the unrelated changes.

You can send them as follow-up PRs